### PR TITLE
Clean up formats in stores topics folder (part 2)

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This was primarily as a fix for #26 (convert tables in stores), but there are other formatting and style issues to address in these topics.

- Converted remaining HTML tables to MD
- Replaced H2/H3 infinitives with present tense verb headings (better for machine translation and findability)
- Adjusted some procedures to better use numbered steps, as well as clarify navigation and simplify the instructions
- Added alt text for images where needed
- A few misc. formats to align with MD linting rules
- Fix for conditioned content where editions are the same
- Fixed TOC/Nav label in yml files to reflect edited page title

**_Note:_** To keep the PR from being too large, this is the second of a series to address #26

## Affected documentation pages

https://docs.magento.com/m2/b2b/user_guide/stores/attributes-input-types.html
https://docs.magento.com/m2/b2b/user_guide/stores/attributes-product.html
https://docs.magento.com/m2/b2b/user_guide/stores/attributes-rating.html
https://docs.magento.com/m2/b2b/user_guide/stores/attributes-returns.html
https://docs.magento.com/m2/b2b/user_guide/stores/attributes.html
https://docs.magento.com/m2/b2b/user_guide/stores/b2b-basics.html
https://docs.magento.com/m2/b2b/user_guide/stores/b2b-best-practices.html
https://docs.magento.com/m2/b2b/user_guide/stores/b2b-configuration-settings.html
https://docs.magento.com/m2/b2b/user_guide/stores/b2b-configure-quotes.html
https://docs.magento.com/m2/b2b/user_guide/stores/b2b-features.html
https://docs.magento.com/m2/b2b/user_guide/stores/compliance-cookie-law.html
https://docs.magento.com/m2/b2b/user_guide/stores/compliance-cookie-restriction-mode.html
https://docs.magento.com/m2/b2b/user_guide/stores/compliance-gdpr-google.html
https://docs.magento.com/m2/b2b/user_guide/stores/compliance-gdpr.html
https://docs.magento.com/m2/b2b/user_guide/stores/compliance-industry.html
https://docs.magento.com/m2/b2b/user_guide/stores/compliance-pci.html
https://docs.magento.com/m2/b2b/user_guide/stores/configuration-overview.html
https://docs.magento.com/m2/b2b/user_guide/stores/configuration.html
https://docs.magento.com/m2/b2b/user_guide/stores/contact-us.html
https://docs.magento.com/m2/b2b/user_guide/stores/cookie-reference.html
https://docs.magento.com/m2/b2b/user_guide/stores/country-options.html
https://docs.magento.com/m2/b2b/user_guide/stores/currency-configuration.html
https://docs.magento.com/m2/b2b/user_guide/stores/currency-overview.html
https://docs.magento.com/m2/b2b/user_guide/stores/currency-symbols.html
https://docs.magento.com/m2/b2b/user_guide/stores/currency-update.html
https://docs.magento.com/m2/b2b/user_guide/stores/currency.html
https://docs.magento.com/m2/b2b/user_guide/stores/locale-options.html
https://docs.magento.com/m2/b2b/user_guide/stores/privacy-policy.html
https://docs.magento.com/m2/b2b/user_guide/stores/security-admin.html
https://docs.magento.com/m2/b2b/user_guide/stores/security-browser-capabilities-detection.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

Internal staging build 234

### CODE REVIEW

File: src/stores/attributes-product.md [hunk 1]
```diff
@@ -2,152 +2,60 @@
 title: Product Attributes
 ---
 
-Product attribute properties determine how an attribute can be used in the catalog, its appearance and behavior in the store, and in data transfer operations. The properties and labels associated with each attribute determine how it can be used, and its presentation in the store.
+Product attribute properties determine how an attribute can be used in the catalog, its appearance and behavior in the store, and the data included in data transfer operations. The properties and labels associated with each attribute determine how it can be used and its presentation in the store.
```
Comment (dobooth) [inline src/stores/attributes-product.md]: None of these tables seem to be rendering properly...at least in Github

Comment (dobooth) [reply src/stores/attributes-product.md]: Wonder if that property-table attribute is breaking it...

File: src/stores/attributes-returns.md [hunk 2]
```diff
@@ -6,18 +6,16 @@ group: sales
...
 _Returns Attributes_
 
-## To create a returns attribute:
+## Create a returns attribute
 
-1.  On the _Admin_ sidebar, choose **Stores**.
+1. On the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Returns**.
 
-1.  In the _Attributes_ section, choose **Returns**.
+1. In the upper-right corner, click **Add New Attribute**
```
Comment (dobooth) [inline src/stores/attributes-returns.md]: Missing period

File: src/stores/compliance-pci.md [hunk 3]
```diff
@@ -3,89 +3,24 @@ title: PCI Compliance Guidelines
...
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Maintain a policy that addresses information security.</td>
-         </tr>
-      </tbody>
-   </table>
-To learn more, see: [Magento Approach to PCI Compliance][1].
+|![checkbox]({{ site.baseurl }}{% link images/images/assets/checkbox.png %})|Install and maintain a firewall configuration to protect cardholder data.|
```
Comment (dobooth) [inline src/stores/compliance-pci.md]: This table isn't rendering in the Github preview.

File: src/stores/attributes-product.md (hunk 1 above)
Comment (leslietilling) [reply src/stores/attributes-product.md]: Verified this in staging build 234. The property-table attribute is probably the issue in the GitHub rendering, but necessary to fix the issue with auto sizing and some of these extra long field names.

File: src/stores/compliance-pci.md (hunk 3 above)
Comment (leslietilling) [reply src/stores/compliance-pci.md]: This one has checkbox images in the first column, and GitHub does not like that. But is does render correctly in the staging build.

File: src/stores/attributes-returns.md (hunk 2 above)
Comment (leslietilling) [reply src/stores/attributes-returns.md]: fixed

**Comment by leslietilling** [comment]
running tests

**Pull request merged**